server/images.go

```diff
@@ -48,16 +58,42 @@ type Model struct {
 	ParentModel    string
 	AdapterPaths   []string
 	ProjectorPaths []string
-	Template       string
 	System         string
 	License        []string
 	Digest         string
 	Options        map[string]interface{}
 	Messages       []Message
+
+	Template *template.Template
 }
 
-func (m *Model) IsEmbedding() bool {
-	return slices.Contains(m.Config.ModelFamilies, "bert") || slices.Contains(m.Config.ModelFamilies, "nomic-bert")
+func (m *Model) Can(caps ...Capability) bool {
```

Comment:
Does this check over all "model" layers for a given model manifest? Generally there should only be one but thought it's worth considering

Reply:
there can only be a single "model". there maybe multiple in the Modelfile but by this point, only the last occurrence is considered